CREDITS.md

```diff
@@ -195,17 +212,21 @@ Contributors to the Firecracker repository:
 - Sam Jackson <sam@clique.app>
 - Samuel Knox <knoxsa@oregonstate.edu>
 - Samuel Ortiz <sameo@linux.intel.com>
+- ScmTble <ScmTble@qq.com>
 - Sean Lavine <freewil@users.noreply.github.com>
 - Sebastien Boeuf <sebastien.boeuf@intel.com>
 - Serban Iorga <seriorga@amazon.com>
+- ShadowCurse <yegorlz@amazon.co.uk>
```

Comment:
Also seems can be de-duplicated ;)

Reply:
Names are different, so I don't think this can be fixed.

Reply:
But the email is the same. You just need an entry in mailmap like this:

```
Egor Lazarchuk <yegorlz@amazon.co.uk>
```

Reply:
oh, ok will fix

Reply:
fixed